react/features/app-settings/components/AppSettings.native.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```

Comment:
Please add `// @flow` to this file too. Unfortunately it will point out a few things that need to be taken care of.

Reply:
Done, thanks!

@jitsi/mobile can you please take a look? It seems so that this is the first native class with a superclass and with flow enabled. Had to declare all super methods manually to avoid errors.

Reply:
https://github.com/zbettenbuk/jitsi-meet/blob/1e8ec9da2e42fca34d122286f4389fed4f9e3461/react/features/app-settings/components/AppSettings.native.js#L193

Reply:
https://github.com/zbettenbuk/jitsi-meet/blob/1e8ec9da2e42fca34d122286f4389fed4f9e3461/react/features/app-settings/components/AppSettings.native.js#L191